ghedesigner/media.py

```diff
@@ -1,58 +1,87 @@
-from dataclasses import InitVar, asdict, dataclass, field
-
-from pygfunction.media import Fluid
+from scp.ethyl_alcohol import EthylAlcohol
+from scp.ethylene_glycol import EthyleneGlycol
+from scp.methyl_alcohol import MethylAlcohol
+from scp.propylene_glycol import PropyleneGlycol
+from scp.water import Water
 
 from ghedesigner.enums import FluidType
 
 
-@dataclass
-class GHEFluid(Fluid):
```

Comment:
@axelstudios I didn't understand the `@dataclass` stuff here, and pulling it out didn't seem to break anything. We can discuss whether any fixups are needed later.